test/e2e/common/node/runtime.go

```diff
@@ -168,9 +168,9 @@ while true; do sleep 1; done
 				gomega.Expect(c.Delete()).To(gomega.Succeed())
 			}
 
-			ginkgo.It("should report termination message [LinuxOnly] if TerminationMessagePath is set [NodeConformance]", func() {
-				// Cannot mount files in Windows Containers.
-				// TODO(claudiub): Remove [LinuxOnly] tag once Containerd becomes the default
+			ginkgo.It("should report termination message if TerminationMessagePath is set [Excluded:WindowsDocker] [NodeConformance]", func() {
+				// Cannot mount files in Windows Docker Containers.
```

Comment:
What does the addition of word "Docker" signifies here? Built by Docker?

Reply:
I think this means created by docker.

Reply:
is it important how the container was created? Just out of curiosity

Reply:
```suggestion
				// Cannot mount files in Windows Containers created by Docker.
```

Reply:
Yes. 
Docker only uses HCSv1 APIs and those APIs can only mount in entire directories.
ContainerD uses HCSv2 APIs which can mount in individual files.
Also Docker/Moby on Windows has not been updated to use ContainerD as the container runtime :(